CONTRIBUTING.md

```
@@ -0,0 +1,341 @@
+- [ ] Follows style guidelines
+- [ ] Documentation updated (if applicable)
+- [ ] Commit messages follow convention
+- [ ] PR description is clear and complete
+
+## Style Guidelines
+
+### Commit Messages
+
+Follow [Conventional Commits](https://www.conventionalcommits.org/):
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)